--- liboop-1.0/configure.ac	2003-10-26 08:39:21.000000000 -0800
+++ liboop-1.0/configure.ac.new	2008-12-04 11:24:54.000000000 -0800
@@ -71,7 +71,7 @@
 fi
 
 if test xno != x$with_tcl; then
-  for version in 8.4 8.3 8.2 8.1 8.0 ; do
+  for version in 8.6 8.5 8.4 8.3 8.2 8.1 8.0 ; do
     CPPFLAGS="$save_cppflags -I/usr/include/tcl$version"
     AC_CHECK_LIB(tcl$version,Tcl_Main,[
     AC_CHECK_HEADER(tcl.h,[
